sockeye/training.py

```diff
@@ -1061,7 +1077,10 @@ def log_metrics(self, metrics: Dict[str, Union[float, int, mx.nd.NDArray]], chec
 
         for name, value in metrics.items():
             if isinstance(value, mx.nd.NDArray):
-                self.sw.add_histogram(tag=name, values=value, bins=100, global_step=checkpoint)
+                if mx.ndarray.contrib.isfinite(value).sum() == value.size():
```

Comment:
when was this added to mxnet? It does not seem to be present in my version (`1.3.1`).

Reply:
It's mentioned in the 1.4.0 changelog: https://github.com/apache/incubator-mxnet/blob/master/NEWS.md So maybe we  should use numpy for now and add a TODO to use the mxnet version with the next release.

Reply:
Or simply add the three line function to `utils.py` for now https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/ndarray/contrib.py#L491